### DISCUSSION TITLE
Always on Top / PiP Mode

### DISCUSSION THREAD

**Discussion by Cronnie11** [f1multiviewer/issue-tracker#21] [Feature requests] [answered]
First off just wanted to say fantastic work on the app so far, the app has a great foundation can't wait to see where it goes from here.

I think a built-in option to toggle windows to be always on top would be a great addition, it would allow PiP functionality if users want to set certain windows over apps they may be actively using.

**Comment by f1multiviewer** [comment, marked as answer]
This is available now! Right clicking on the feed allows you to mark it as "always on top"
